playlistmanager.lua

```diff
@@ -489,7 +492,21 @@ end
 function draw_playlist()
   refresh_globals()
   local ass = assdraw.ass_new()
-  ass:new_event()
+	
+  local _, _, a = mp.get_osd_size()
+  local h = 360
+  local w = h * a
+
+  if settings.curtain_opacity ~= nil and settings.curtain_opacity < 1.0 then
```

Comment:
Can we add a check here that opacity ~= 0.

I'd like the default setting of `0.0` to not go inside this branch.

Reply:
I think I changed it to how you want

Reply:
still need to check whether it is nil